concent_api/core/message_handlers.py

```diff
@@ -1354,6 +1366,9 @@ def handle_send_subtask_results_verify(
             reason=message.concents.ServiceRefused.REASON.InvalidRequest,
         )
 
+    if Subtask.objects.filter(subtask_id=compute_task_def['subtask_id']).exists():  # pylint: disable=no-member
+        Subtask.objects.select_for_update().get(subtask_id=compute_task_def['subtask_id'])  # pylint: disable=no-member
+
```

Comment:
You can replace these three queries with one. Same as in one of my comments above.

Reply:
done